### PR TITLE
Translate program details

### PR DESCRIPTION
**Pull request by jbaldo**
Translations for the program details page. Rebased with Kabirdas's work.

Screen Shot 2020-12-17 at 5 43 23 PM

### CODE REVIEW

File: src/components/central-program/program-data-overview-table.js [hunk 1]
```diff
@@ -3,28 +3,46 @@ import PropTypes from "prop-types"
...
+const DESCRIPTION = "description"
+const VALUE = "value"
+const STAFF = "staff"
+
+const CHANGE = "change_from_previous_year"
+const SPENDING = "spending"
+const BUDGET = "budget"
+const BALANCE = "budget_balance"
+const STAFF_FTE = "eoy_total_fte"
+const STAFF_POSITIONS = "eoy_total_positions"
```
Comment (jbaldo) [inline src/components/central-program/program-data-overview-table.js:17]: Not sure if these constants make sense. Was just trying to put some distance from the names in Contentful and all the places where we reference them below.

File: src/components/central-program/program-data-overview-table.js [hunk 2]
```diff
@@ -237,14 +255,17 @@ const StaffOverview = ({ data }) => {
...
+  columnsByDataField = getColumnsByDataField(content.columns)
+  return (
+    <div className={`program-data-overview ${className}`}>
+      <Heading />
+      <SpendingOverview data={data.centralProgramsJson} />
+      <BudgetOverview data={data.centralProgramsJson} />
+      <StaffOverview data={data.centralProgramsJson} />
+    </div>
+  )
+}
```
Comment (jbaldo) [inline src/components/central-program/program-data-overview-table.js:268]: I got `prettier` working locally, so some of this is auto-reformatting.

File: src/components/central-program/program-data-overview-table.js [hunk 3]
```diff
@@ -139,25 +141,25 @@ const BudgetOverview = ({ data }) => {
...
     overOrUnder = "Under"
-    difference = data[BUDGET] - data[SPENDING]
+    balance = data[BUDGET] - data[SPENDING]
   }
 
   rows.push({
-    description: "Over or under budget?",
-    value: `${overOrUnder} by ${formatToUSD(difference)} (${Math.abs(
+    description: columnsByDataField[BALANCE].displayName,
+    value: `${overOrUnder} by ${formatToUSD(balance)} (${Math.abs(
```
Comment (kbrdsk) [inline src/components/central-program/program-data-overview-table.js:162]: "Over/Under by " here isn't being translated, maybe add an overOrUnder property columnsByDataField.budget_balance?

File: src/components/central-program/program-data-overview-table.js (hunk 1 above)
Comment (emilybram) [reply src/components/central-program/program-data-overview-table.js:17]: seems reasonable to me!

**Review by emilybram — APPROVED** [review]
this looks good to me!

File: src/components/central-program/program-data-overview-table.js (hunk 3 above)
Comment (jbaldo) [reply src/components/central-program/program-data-overview-table.js:162]: thanks. yeah, I need to tidy that up. wasn't exactly sure what the translation should be but I'll pick something and can always come back and adjust.

**Pull request merged**